src/data/data.ts

```diff
@@ -0,0 +1,1908 @@
+export interface IProduct {
```

Comment:
A data file is not a good place for type declarations

Reply:
fixed   
[7afdd23](https://github.com/AP-Est/Online-Store/pull/65/commits/7afdd23580f276daee941c235f77627c1d0ee9fc)

Reply:
Make additions -> aa95567a79903ef279b6adfc0bba0ef4f822750b